src/Microsoft.Health.Fhir.SqlServer/Features/Operations/Import/ImportOrchestratorJob.cs

```diff
@@ -264,9 +272,33 @@ private async Task ValidateResourcesAsync(ImportOrchestratorJobDefinition inputD
             });
         }
 
-        private async Task SendImportMetricsNotification(JobStatus jobStatus, JobInfo jobInfo, ImportOrchestratorJobResult currentResult, ImportMode importMode)
+        private async Task SendImportMetricsNotification(JobStatus jobStatus, JobInfo jobInfo, ImportOrchestratorJobResult currentResult, ImportMode importMode, FhirRequestContext fhirRequestContext)
         {
             _logger.LogInformation("SucceededResources {SucceededResources} and FailedResources {FailedResources} in Import", currentResult.SucceededResources, currentResult.FailedResources);
+
+            if (importMode == ImportMode.IncrementalLoad)
+            {
+                var incrementalImportProperties = new Dictionary<string, string>();
+                incrementalImportProperties["JobId"] = jobInfo.Id.ToString();
+                incrementalImportProperties["SucceededResources"] = currentResult.SucceededResources.ToString();
+                incrementalImportProperties["FailedResources"] = currentResult.FailedResources.ToString();
+
+                _auditLogger.LogAudit(
+                AuditAction.Executed,
+                operation: ImportMode.IncrementalLoad.ToString(),
+                resourceType: string.Empty,
+                requestUri: fhirRequestContext.Uri,
+                statusCode: HttpStatusCode.Accepted,
+                correlationId: fhirRequestContext.CorrelationId,
+                callerIpAddress: string.Empty,
+                callerClaims: null,
+                customHeaders: null,
+                operationType: string.Empty,
+                callerAgent: DefaultCallerAgent,
+                additionalProperties: incrementalImportProperties);
```

Comment:
For my understanding, based on this field in health-paas -> FhirOperationAuditLogger we will determine to only write to shoebox log?

Reply:
For incremental import properties, we will be writing only to Shoebox logs. But we can use this field for any other values as well. In future, if we have a requirement to send any additional information, we can use this field.

Reply:
Right, so currently whenever the [logAudit](https://microsofthealth.visualstudio.com/Health/_search?action=contents&text=fhiroperationauditlogger&type=code&lp=code-Project&filters=ProjectFilters%7BHealth%7DRepositoryFilters%7Bhealth-paas%7D&pageSize=25&result=DefaultCollection/Health/health-paas/GBmain//src/FhirApplication/FhirService.Core/Features/AzureSecurityAudit/FhirOperationAuditLogger.cs) method is invoked, logged at both places(shoebox and azsecurity). I was wondering what check we will add in order to just invoke shoebox logger and not azSecurity.

Reply:
I understand your point now. Based on current changes, we are explicitly calling both loggers. I can add a check on HP side to check for operation and skip this from security logs. Honestly, I do not see why we cannot log this to both security audit and shoebox logs. We do not have any customer sensitive information in this log. I think I will add it to both logs and I will confirm with Ketki as well.

Reply:
Yes lets confirm with Ketki. Otherwise we should be good putting a check in the [AzureSecurityAuditLogger](https://microsofthealth.visualstudio.com/Health/_search?action=contents&text=azuresecurityauditlogger&type=code&lp=code-Project&filters=ProjectFilters%7BHealth%7DRepositoryFilters%7Bhealth-paas%7D&pageSize=25&result=DefaultCollection/Health/health-paas/GBmain//src/FhirApplication/FhirService.Core/Features/AzureSecurityAudit/AzureSecurityAuditLogger.cs) itself at Ln66